src/transformers/modeling_flax_utils.py

```diff
@@ -60,19 +60,24 @@ def gelu(x):
 
 
 class FlaxPreTrainedModel(ABC):
+    config = None
     config_class = None
     pretrained_model_archive_map = {}
     base_model_prefix = ""
-    model_class = None
 
     def __init__(
-        self, config: PretrainedConfig, module: nn.Module, params: Dict, seed: int = 0, dtype: jnp.dtype = jnp.float32
+        self,
+        config: PretrainedConfig,
+        module: nn.Module,
+        params: Optional[Dict] = None,
+        seed: int = 0,
+        dtype: jnp.dtype = jnp.float32,
     ):
         if config is None:
             raise ValueError("config cannot be None")
 
-        if params is None:
```

Comment:
To randomely initialize a model and allow `BertModel(config)` I think we shouldn't do this check

Reply:
Yap, better removing this check.